### ISSUE TITLE
question

### ISSUE THREAD

**Issue by alexuk79** [FaxHack/Cracked.io-ServerEmulator#1]
hi, this is still working? can it be detected by cracked.io?

**Comment by FaxHack** [comment]
yes its still working to this day and no it cant be detected by cracked.io themselves.

**Comment by Tibosity** [comment]
ok so i ran this and now i cant even access cracked.io whatsoever. 
![image_2023-07-11_110830557](https://github.com/FaxHack/Cracked.io-ServerEmulator/assets/89561185/d819a303-7f37-471b-a0f1-35a402e07370)

**Comment by FaxHack** [comment]
> ok so i ran this and now i cant even access cracked.io whatsoever. 
> 
> ![image_2023-07-11_110830557](https://github.com/FaxHack/Cracked.io-ServerEmulator/assets/89561185/d819a303-7f37-471b-a0f1-35a402e07370)
> 
> 

When you want to exit the emulator you need to make sure to press enter don't shut it off

**Comment by Tibosity** [comment]
ok well now it's doing this
![opera_LwyQahXurN](https://github.com/FaxHack/Cracked.io-ServerEmulator/assets/89561185/bc0b4596-9a29-441d-9f60-f2221ddab5ae)

**Comment by Tibosity** [comment]
nevermind, i fixed it.

**Comment by Tibosity** [comment]
i turned on my pc and now it only lets me visit the cracked.io home page, whenever i go to any other page it shows this
![opera_hiojjwP3zN](https://github.com/FaxHack/Cracked.io-ServerEmulator/assets/89561185/1c1d9422-794f-4dbe-9346-10d9c2c6c6f2)
![opera_VozHvKKFmx](https://github.com/FaxHack/Cracked.io-ServerEmulator/assets/89561185/bb36fd4c-513a-4ddd-8801-eb9f5bb7bf5a)

**Comment by efsanevidemir** [comment]
![Adsız](https://github.com/FaxHack/Cracked.io-ServerEmulator/assets/83442695/1255f9fc-73f9-44c6-b0bc-52edb9455324)
What exactly is the problem buddy? can you help me? I did all the steps you said in order.

**Comment by efsanevidemir** [comment]
![Adsız](https://github.com/FaxHack/Cracked.io-ServerEmulator/assets/83442695/99ed2cc2-c57e-4ed1-82c5-0ea8d3563d1c)

Dude, why does it look like this on me? When I log in to my account, it is not detected.

**Comment by FaxHack** [comment]
what tool did you try using?

**Comment by efsanevidemir** [comment]
Brave Browser

**Comment by FaxHack** [comment]
I said tool not browser

**Comment by efsanevidemir** [comment]
> I said tool  not browser

What do you mean by Tool, my friend? I just did the steps you said and opened it
